software/NRG_itho_wifi/TaskMQTT.ino

```diff
@@ -26,6 +26,8 @@ void TaskMQTT( void * pvParameters ) {
     vTaskDelay(25 / portTICK_PERIOD_MS);
   }
 
+  execAutoPilot();
```

Comment:
This call is actually outside the loop and will never be called

Reply:
and this kids is why you don't debug after a long day late at night :blush: